modules.d/95fcoe/lldpad.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-if getargbool 0 rd.nofcoe ; then
-	info "rd.nofcoe=0: skipping lldpad activation"
+if ! getargbool 1 rd.fcoe -d rd.nofcoe ; then
```

Comment:
Shall we follow the wrong documentation with the deprecated option or the correct one with:

```suggestion
if ! getargbool 1 rd.fcoe -d -n rd.nofcoe ; then
```